### PR TITLE
Fix lp:1702903 (Cannot build Percona Server on Debian Stretch) (5.7)

### PR DESCRIPTION
**Pull request by percona-ysorokin**

### CODE REVIEW

**Comment by percona-ysorokin** [comment]
Jenkins build links:
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1049 (OpenSSL)
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1050 (YaSSL)

File: mysql-test/t/ssl_bug75311.test [hunk 1]
```diff
@@ -14,7 +14,7 @@ SHOW STATUS LIKE 'Ssl_cipher_list';
 --connection default
 
 # The first error string is returned by YaSSL, the second one by OpenSSL
```
Comment (laurynas-biveinis) [inline mysql-test/t/ssl_bug75311.test]: Three replacements now but the comment explains only the two of them (this comment applies to lower versions too)

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
+ same question re. EC ciphers in the testsuite as in 5.6

**Review by george-lorch — APPROVED** [review]
Same thing as 5.6, you might want to reload the failed jenkins platforms just to be sure.

File: mysql-test/t/ssl_bug75311.test (hunk 1 above)
Comment (percona-ysorokin) [reply mysql-test/t/ssl_bug75311.test]: Fixed both in 5.6 and 5.7

**Comment by percona-ysorokin** [comment]
EC- cipher testsuite changes were indeed not necessary. They were added just for symmetry reasons. Reverted.

**Comment by percona-ysorokin** [comment]
Updated Jenkins build links
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1065 (OpenSSL)
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1066 (YaSSL)

**Comment by percona-ysorokin** [comment]
@laurynas-biveinis @georgelorch  As there were no code changes (only EC-ciphers fixes and comment corrections in the MTR test cases) since the last review and Jenkins builds are much more stable now, I am going to merge this PR.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**